### DISCUSSION TITLE
CarPlay swipe guestures

### DISCUSSION THREAD

**Discussion by imixerpro** [organicmaps/organicmaps#3431] [Ideas]
Speaking about CarPlay: panning guestures (swipes to move the map) aren’t implemented for some reason.

Being able to navigate to arbitrary point (I know there no generic touches, map view center is ok if panning guestures are implemented will do the job, I think) is also a really good option. This approach is used in my car’s native navigation app.

My car has touchscreen which supports panning guestures and maps.me supports them (or  mapy.cz , not sure).

**Comment by biodranik** [comment]
Thanks! Created an issue for it: #3482
